source/plugin/tab-lists.rst

```diff
@@ -0,0 +1,107 @@
+=========
+Tab Lists
+=========
+
+Tab lists are used in Minecraft to display the list of players currently on a server. The SpongeAPI allows for
+manipulation of the tab list on a per-player basis.
+
+To get a player's ``TabList``, you simply need to call the ``getTabList()`` method off of the ``Player`` whose
+``TabList`` you wish to modify:
+
+.. code-block:: java
+    
+    import org.spongepowered.api.entity.living.player.Player;
+    import org.spongepowered.api.entity.living.player.tab.TabList;
+    
+    TabList tablist = player.getTabList();
+
+Now that we have obtained the ``TabList``, we can modify several components of it. For example, to set the header or
+the footer of the ``TabList``, we simply need to call their appropriate methods:
+
+.. code-block:: java
+    
+    import org.spongepowered.api.text.Text;
+    import org.spongepowered.api.text.format.TextColors;
+    
+    tablist.setHeader(Text.of(TextColors.GOLD, "The tab list header"));
+    tablist.setFooter(Text.of(TextColors.RED, "The tab list footer"));
+
+We can call the ``setHeaderAndFooter()`` method if we want to alter both of them at once:
+
+.. code-block:: java
+    
+    tablist.setHeaderAndFooter(Text.of("header"), Text.of("footer"));
+
+.. note::
+    
+    If you are wanting to alter the tab list header *and* footer, it is recommended to use the ``setHeaderAndFooter()``
+    method over individually calling the ``setHeader()`` and ``setFooter`` methods, as it only sends one packet instead
+    of two separate packets for the header and the footer.
+
+Tab List Entries
+================
+
+Now that we have set the header and footer of the ``TabList``, we can also add our own entries to the list. An example
+of this is shown below:
+
+.. code-block:: java
+    
+    import org.spongepowered.api.entity.living.player.gamemode.GameModes;
+    import org.spongepowered.api.entity.living.player.tab.TabListEntry;
+    import org.spongepowered.api.profile.GameProfile;
+    
+    TabListEntry entry = TabListEntry.builder()
+        .list(tablist)
+        .gameMode(GameModes.SURVIVAL)
+        .profile(gameProfile)
+        .build();
+    tablist.addEntry(entry);
+
+Now let's break this down. We set the ``TabListEntry`` to our specified ``TabList`` using the ``list()`` method. We
```

Comment:
You're not setting the `TabListEntry` to the `TabList`, you're setting the list _associated_ with the `TabListEntry` to the `TabList`